source/unified-test-format/schema-1.2.json

```diff
@@ -0,0 +1,307 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
See [DRIVERS-1485](https://jira.mongodb.org/browse/DRIVERS-1485) and https://github.com/mongodb/specifications/pull/895. A new minor version should rename the existing schema file and then incorporate any relevant changes. Doing so should result in this showing up as a rename/modified diff instead of a new file.